scripts/copyGitHub.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * This module copies the Enact libraries into the `raw` directory as a side effect of loading it.
+ * It will not copy if the libraries already exist, unless the `--force-github` argument is supplied.
```

Comment:
With a name change to `prepareRaw`, the `--force-github` should probably change too. Maybe something like `--force-prepare-raw` or `--rebuild-raw` or something like that.